pynestml/codegeneration/nest_code_generator.py

```diff
@@ -747,6 +755,16 @@ def _get_neuron_model_namespace(self, neuron: ASTNeuron) -> Dict:
                                                       if isinstance(sym.get_type_symbol(), (UnitTypeSymbol, RealTypeSymbol))
                                                       and sym.is_recordable]
 
+        namespace["use_gap_junctions"] = self.get_option("gap_junctions")["enable"]
+        if namespace["use_gap_junctions"]:
+            namespace["gap_junction_membrane_potential_variable"] = self.get_option("gap_junctions")["membrane_potential_variable"]
+
+            var = ASTUtils.get_state_variable_by_name(neuron, self.get_option("gap_junctions")["membrane_potential_variable"])
+            namespace["gap_junction_membrane_potential_variable_is_numeric"] = "_is_numeric" in dir(var) and var._is_numeric
+
+            namespace["gap_junction_membrane_potential_variable_cpp"] = NESTVariablePrinter(expression_printer=None).print(var)
```

Comment:
Could this be moved to the template?

Reply:
I suppose the template shouldn't try to instantiate printers, but we could feed this custom ``NESTVariablePrinter `` instance into the namespace instead as "gap_junction_membrane_potential_variable_printer_cpp" or so.